src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -32,12 +32,23 @@ function LtiConfigForm({
   const isInvalidConsumerSecret = touched.consumerSecret && errors.consumerSecret;
   const isInvalidLaunchUrl = touched.launchUrl && errors.launchUrl;
 
+  const handleValidation = (event) => {
+    event.preventDefault();
+    validateForm().then(() => {
+      handleConfigError(true);
```

Comment:
We're saying there's an error whenever the form is validated?  That seems odd... Doesn't that mean it'll show "Incomplete" whenever the form is submitted?